nvflare/fuel/f3/stats_pool.py

```diff
@@ -12,15 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import csv
+import json
 import threading
 
 from nvflare.fuel.utils.stats_utils import CounterPool, HistPool, new_message_size_pool, new_time_pool
 
 
 class StatsPoolManager:
```

Comment:
A general question, since this file solely depends on `nvflare.fuel.utils.stats_utils`.
And does not need things from f3, should we move this file into nvflare/fuel/utils?

And since this file is about "StatsPoolManager" but not "StatsPool" should we rename it to "stats_pool_manager.py"?

Reply:
Maybe. I do feel that the stats_pool and stats_utils could be merged into one thing. 
Let's do this later.